activerecord/lib/active_record/connection_adapters/sqlite3/schema_statements.rb

```diff
@@ -2,6 +2,41 @@ module ActiveRecord
   module ConnectionAdapters
     module SQLite3
       module SchemaStatements # :nodoc:
+        # Returns an array of indexes for the given table.
```

Comment:
Can we put this method back where it was? I know it's technically supposed to be here, but it shouldn't be moved in this commit/PR (unrelated to adding support for something in MySQL)